.github/workflows/check_precompiled.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Pass updated files in $updated_files environment variable
+
+# enc_bootloader
+echo "$updated_files" | grep -q "^enc_bootloader/.*\.elf$"
+enc_bootloader_elf_not_updated=$?
+if [ $enc_bootloader_elf_not_updated -eq 1 ]; then
+    echo "Checking enc_bootloader files for modifications as ELFs have not been updated"
+    for file in enc_bootloader/*; do
+        if [[ "$file" == "enc_bootloader/CMakeLists.txt" || "$file" == "enc_bootloader/BUILD.bazel" ]]; then
+            continue
+        fi
+        if echo "$updated_files" | grep -q "$file"; then
```

Comment:
If you wanted to avoid the use of all these `grep -q` calls, I guess you could use the [builtin bash functionality](https://stackoverflow.com/a/229606) instead (and there's also a [regex-version](https://stackoverflow.com/a/231298) that you could use on e.g. line 6 above).

Reply:
Hmmm, perhaps the `^` and `$` are matching start-of-string and end-of-string, rather than start-of-line and end-of-line?
Doesn't really matter though, keep using your existing solution :slightly_smiling_face: